lib/ddtrace/contrib/lograge/instrumentation.rb

```diff
@@ -0,0 +1,38 @@
+module Datadog
+  module Contrib
+    module Lograge
+      # Instrumentation for Lograge
+      module Instrumentation
+        def self.included(base)
+          base.prepend(InstanceMethods)
+        end
+
+        # Instance methods for configuration
+        module InstanceMethods
+          def custom_options(event)
+            original_custom_options = super(event)
```

Comment:
Will users be able to override `dd: ...` fields if they need to?
I'm not 100% sure how `custom_options` ordering works, but I think we should set our `dd: ...` fields, but allow `custom_options` to override them, it possible.

Reply:
yea, i'll switch thiis around, so custom_options can clobber `dd` fields if they prefer for whatever reason

Reply:
updated